code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -161,13 +161,43 @@
 	density = FALSE
 	store_mobs = FALSE
 	wall_mounted = TRUE
-
+	var/hijack = FALSE
 
 /obj/structure/closet/secure_closet/surgical/Initialize()
 	. = ..()
 	new /obj/item/storage/surgical_tray(src)
 	new /obj/item/roller/surgical(src)
 
+/obj/structure/closet/secure_closet/surgical/emergency
+	name = "emergency surgical equipment cabinet"
+	desc = "A self-sterilizing, wall-mounted cabinet containing extra surgical beds and empty surgical webbing vests for doctors who suddenly need to operate outside of medbay. Only the CMO can unlock this, but in dire emergencies, it unlocks for the entire medbay."
+	req_access = list(ACCESS_MARINE_CMO)
+
+/obj/structure/closet/secure_closet/surgical/emergency/Initialize()
+	. = ..()
+	new /obj/item/storage/internal/accessory/surg_vest(src)
+	new /obj/item/storage/internal/accessory/surg_vest(src)
+	new /obj/item/roller/surgical(src)
+	new /obj/item/roller/surgical(src)
+	RegisterSignal(SSdcs, COMSIG_GLOB_UNSCHEDULED_DROPSHIP_DEPARTURE_DETECTED, PROC_REF(all_docs_are_field_docs))
+	return
+
+/obj/structure/closet/secure_closet/surgical/emergency/togglelock(mob/living/user)
+	if(hijack == FALSE)
+		if(user.job == JOB_DOCTOR_ROLES_LIST) //I lied. Nobody can open this if hijack == FALSE.
+			to_chat(user, SPAN_WARNING("As medical staff, you may only toggle this lock when shipside doctors must operate on the ship and outside the medical bay."))
```

Comment:
I'm not a mantainer, but the CMO being told they have access to this and then the game saying 'No, you CANNOT do it actually!' is very awkward and weird. It'd be better to just say it's fully locked and restricted to shipside emergencies